resources/fetch-kubelet.tmpl

```diff
@@ -0,0 +1,7 @@
+# https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/?origin_team=T0HR00WDA#installing-kubeadm-kubelet-and-kubectl
```

Comment:
Would it be possible to simply use `ignition_file` here? https://www.terraform.io/docs/providers/ignition/d/file.html